tests/kuttl/tests/openstackcontrolplane_cpu_ram_scale/04-assert.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Check for:
+#
+# - verify the VM has 6 cores
+# - verify the VM has 20 GB RAM
+#
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 180
+commands:
+  - script: |
+      #!/usr/bin/env bash
+      sleep 10
+      CONTROLLER=$(oc get -n openstack osnet ctlplane -o json | jq .status.reservations | jq -r "to_entries | map(select(.key == \"controller-0\")) | unique[] | .value.ip")
+      CPUS=$(oc rsh -n openstack openstackclient ssh $CONTROLLER "grep processor /proc/cpuinfo | wc -l")
+      RAM=$(oc rsh -n openstack openstackclient ssh $CONTROLLER grep MemTotal /proc/meminfo | awk '{ print $2 }')
+      if [[ ( "${CPUS}" != 6 ) || ( "${RAM}" != 20340536 ) ]]; then
```

Comment:
I don't think the RAM check will be reliable. The MemTotal value is always less than the physical RAM. I expect it will vary depending on the libvirt/qemu version (bios, video device etc...) and the kernel version on the VM.

Maybe just check that it's 19-20GB and then check that 20-21GB when the RAM has been increased?

Reply:
ack

Reply:
changed the test

Reply:
hrmm, github has not updated the PR